docs/systemd.md

```diff
@@ -44,7 +44,7 @@ Then you can use the script, specifying the binary name as an argument, e.g.:
 ```
 ./gen_systemd_service_file.py tezos-node
 # or
-./gen_systemd_service_file.py tezos-baker-PtLimaPt
+./gen_systemd_service_file.py tezos-baker-PtMumbai
```

Comment:
There's a couple more mentions of `limanet` in this file (lines 66 and 81) that would nice to remove too